README.md

```diff
@@ -1,42 +1,34 @@
-- All examples within [Docs](https://github.com/RumbleMike/ValorantAPI/tree/master/Docs) are directed to the NA Region.
-
-| Region | URL |
-| - | - |
-| North America | https://pd.NA.a.pvp.net/ |
-| Europe | https://pd.EU.a.pvp.net/ |
-| Asia Pacific | https://pd.AP.a.pvp.net/ |
-| Korea | https://pd.KO.a.pvp.net/ |
-
-## Authentication
-Each API Request requires 2 Headers for authentication.
-`X-Riot-Entitlements-JWT` & `Authentication`
-
-# Table of Contents
-This is here to make your life easier, whenever there is a new addition it gets added here. (Categories will be added in the future.)
-Please check out [Getting Started](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/GettingStarted.md) for an easy Python script that replicates RSO and provides your X-Riot-Entitlements-JWT & Authentication Token.
-- [Getting Started](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/GettingStarted.md)
-- User API's
-	- [Player ID](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/PlayerID.md)
-	- [User Balance](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/UserBalance.md)
-	- [Competitive History](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/CompetitiveHistory.md)
-	- [Player MMR](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/PlayerMMR.md)
-	- [Player Store](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/PlayerStore.md)
-	- [Player Inventory | Coming Soon](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/PlayerInventory.md)
-- Information API's
-	- [Story Contract](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/StoryContract.md)
-	- [ID List](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/IDList.md)
-	- [Client Versions](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/ClientVersions.md)
-	- [**Username From ID**](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/GetUserfromID.md)
-- Match Information APIs
-	- [Match History](https://github.com/RumbleMike/ValorantAPI/blob/master/Docs/MatchHistory.md)
-	- [Match Info | Coming Soon]()
+It takes an OOP approach to replication of the client, so that we can more easily build features in the future.
+
+## ValorantClient.py
+
+Built off the AIOHttp library's ClientSession module, which creates async HTTP Client sessions.
+
+Currently supports the below features:
+
+1. Authentication / log in
+2. [View Valorant Store](#tutorial---view-daily-valorant-store))
+
+## Requirements and Installation
+
+1. Ensure you have Python >= 3.6 installed on your machine
+2. Clone this repo
+3. Create + Activate a virtual environment
+4. run `pip install requirements.txt` inside your virtual environment
+
+## Tutorial - View Daily Valorant Store
+
+In order to view your Daily store:
+
+1. run `ValorantStore.py`
+2. Enter your username and password
+
+You'll then see you're store for the day along with what time it expires.
```

Comment:
can we get a screenshot ? remember HR ppl might look, they could understand quicker through pictures